docs/source/components/nodes/color_camera.rst

```diff
@@ -46,6 +56,11 @@ Inputs and Outputs
 into :ref:`NeuralNetwork`. :code:`video` and :code:`still` are both NV12, so are suitable for bigger sizes. :code:`still` image gets created when
 a capture event is sent to the ColorCamera, so it's like taking a photo.
 
```

Comment:
To be consistent with the existing types, add here that `isp` frames are YUV420 planar (aka YU12, IYUV, I420) and can be resized using (ref)`setIspScale`.

And `raw` outputs packed RAW10 bayer data (for other sensors in the future we may need to have RAW12, and the bayer color order depends on the sensor and image orientation, TODO add to `ImgFrame::Type`: `RAW10_BayerBG`, etc ). 
We have here some code for unpacking: https://github.com/luxonis/depthai-experiments/blob/3f1b2b2/gen2-color-isp-raw/main.py#L13-L32